changelog.asc

```diff
@@ -2,6 +2,51 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.19.0 (20241026)
+* Incorporating the work from three Gsoc projects and the work done in the last couple of months, we are proud to present the latest iteration of the best Android app for learning!  
+* We really appreciate the https://opencollective.com/ankidroid[donations], they paid for these features 🤝
+* Features
+** Includes Anki upstream version 24.06.3 with FSRS 0.6.4 
```

Comment:
Could we put the user facing version in brackets here (lifted from the Anki Chsngelog)

Reply:
I hope I understood this correctly.